komodo/lint.py

```diff
@@ -132,6 +131,7 @@ def lint(
                 _komodo_error(
                     err="Failed requirements ",
```

Comment:
The formatting here looks a little bit unintended, as there is a space after "requirements" (but that is not your bug). Should we spend a colon instead since there apparantly always is a  newline after "requirements"?

Reply:
I think we should fix this yes. 👍